src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -739,4 +741,41 @@ export default class DefaultAudioVideoController implements AudioVideoController
       this.meetingSessionContext.signalingClient.resume([streamId]);
     }
   }
+
+  getRemoteVideosAvailable(): { attendeeId: string; externalUserId: string }[] {
+    if (!this.meetingSessionContext.videoStreamIndex) {
+      this.logger.info('meeting session context has no video stream index');
+      return [];
+    }
+    const selfAttendeeId = this.configuration.credentials.attendeeId;
+    const videoStreamIndex = this.meetingSessionContext.videoStreamIndex;
+    return videoStreamIndex.allVideoSendingAttendeesExcludingSelf(selfAttendeeId);
+  }
+
+  addSimulcastUplinkPolicyObserver(observer: SimulcastUplinkObserver): void {
+    if (
+      this.enableSimulcast &&
+      'addObserver' in this.meetingSessionContext.videoUplinkBandwidthPolicy
+    ) {
+      (this.meetingSessionContext.videoUplinkBandwidthPolicy as SimulcastUplinkPolicy).addObserver(
+        observer
+      );
+    } else {
+      this.logger.warn(`tried to add simulcast uplink policy observer on a non-simulcast policy`);
```

Comment:
Yes, I did not had this, until I figured the instantiation of SimulcastPolicy takes place in `audioVideo.start()`, so I thought it would be good to have a warning log to tell if the builder uses simulcast observer before `audioVideo.start()`.

Reply:
Have removed these functions now.